packages/aws-cdk/package.json

```
@@ -108,26 +108,26 @@
     "@aws-cdk/cloudformation-diff": "0.0.0",
     "@aws-cdk/cx-api": "0.0.0",
     "@aws-cdk/region-info": "0.0.0",
-    "@aws-sdk/client-appsync": "3.699.0",
-    "@aws-sdk/client-cloudformation": "3.699.0",
-    "@aws-sdk/client-cloudwatch-logs": "3.699.0",
-    "@aws-sdk/client-codebuild": "3.699.0",
-    "@aws-sdk/client-ec2": "3.699.0",
-    "@aws-sdk/client-ecr": "3.699.0",
-    "@aws-sdk/client-ecs": "3.699.0",
-    "@aws-sdk/client-elastic-load-balancing-v2": "3.699.0",
-    "@aws-sdk/client-iam": "3.699.0",
-    "@aws-sdk/client-kms": "3.699.0",
-    "@aws-sdk/client-lambda": "3.699.0",
-    "@aws-sdk/client-route-53": "3.699.0",
-    "@aws-sdk/client-s3": "3.699.0",
-    "@aws-sdk/client-secrets-manager": "3.699.0",
-    "@aws-sdk/client-sfn": "3.699.0",
-    "@aws-sdk/client-ssm": "3.699.0",
-    "@aws-sdk/client-sts": "3.699.0",
-    "@aws-sdk/credential-providers": "3.699.0",
-    "@aws-sdk/ec2-metadata-service": "3.699.0",
-    "@aws-sdk/lib-storage": "3.699.0",
+    "@aws-sdk/client-appsync": "^3.699.0",
```

Not super important but I think our package.json normally corresponds with the lock file.

```suggestion
    "@aws-sdk/client-appsync": "^3.730.0",
```

---

Sorta the point of having the distinction between a package.json and a lockfile?

I do not care about this 😉